scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -108,110 +115,110 @@ function doCheckPluginsInstalled() {
       if [ "${plugin}" == "org.eclipse" ] || [ "${plugin}" == "org.apache" ] || [ "${plugin}" == "com" ] || [ "${plugin}" == "org" ] || [ "${plugin}" == "net" ]; then
         break
       fi
-      
+
       local new_plugin_dependencies
       new_plugin_dependencies=$(ls -1 "${plugin_dir}"/*"${plugin}".* 2>/dev/null)
-      if [ -n "${new_plugin_dependencies}" ] 
-      then
+      if [ -n "${new_plugin_dependencies}" ]; then
         plugin_dependencies="${plugin_dependencies}\n${new_plugin_dependencies}"
       fi
       plugin=${plugin%.*}
     done
   done
   # Remove plugin_dir from plugin variable
   plugin_dependencies=$(echo -e "${plugin_dependencies}" | sed "s|${plugin_dir}/||g")
-  echo -e "${plugin_dependencies}"
+  echo -e "${plugin_dependencies}" | sort -u
+}
+
+# Remove plugins from Eclipse by feature group ID
+function doRemovePlugin() {
+  if [ -z "${1}" ]; then
+    doError "No plugin was specified for removal."
+    return 1
+  fi
+  doLoadPluginProperties "${1}"
+  if [ -z "${plugin_id}" ]; then
+    doError "Plugin '${1}' is not installed.\nMake sure you provided the correct plugin name (check settings/eclipse/plugins folder)."
+    return 1
+  else
+    doEcho "Removing plugin ${1}..."
+
+    local debug_info_file
+    debug_info_file=$(mktemp)
+    "${ECLIPSE}" -nosplash -clean -consoleLog -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.equinox.p2.director -uninstallIU "${plugin_id}" -vmargs "${ECLIPSE_VMARGS}" >"${debug_info_file}" 2>&1
+    local exit_code=$?
+    # Overwrite file with lines containing !MESSAGE
+    sed -i -n '/!MESSAGE/ p' "${debug_info_file}"
```

Comment:
You already wrote this:
> Only thing I havent done yet is
"Instead of writing the entire log and then filtering it with sed, we should directly use a pipe for that as already done in line 67 (IMHO we should even create a function for the invocation of eclipse with the -application option, etc. including to write a log file and grepping for an expected message to check if this was successful or not instead of copy&paste)"
this because I am not sure how to do it